package.json

```diff
@@ -143,13 +143,13 @@
     "sass-loader": "7.3.1",
     "size-limit": "^1.3.6",
     "sort-by": "^1.2.0",
-    "standard-version": "^4.4.0",
+    "standard-version": "^7.1.0",
```

Comment:
@greg-a-smith Do you remember if it was on the standard-version end that the api changed for making our release notes? Is this upgrade ok?